src/mixins/filing-template-mixin.ts

```diff
@@ -90,6 +91,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
   @Action setNewResolutionDates!: ActionBindingIF
   @Action setFileNumber!: ActionBindingIF
   @Action setHasPlanOfArrangement!: ActionBindingIF
+  @Action setResolution!: ActionBindingIF
```

Comment:
It would be nice if the action (and mutation) was named similarly to the getter.